### PR TITLE
Create file to list all buckets deleted 

### PR DESCRIPTION
**Pull request by OdedViner**
I want to create this file for the artifact:
image

https://url.corp.redhat.com/fb4f6ea

### CODE REVIEW

File: ocs_ci/cleanup/aws/cleanup.py [hunk 1]
```diff
@@ -324,9 +324,12 @@ def delete_buckets(bucket_prefix, hours):
     aws = AWS()
     buckets_to_delete = aws.get_buckets_to_delete(bucket_prefix, hours)
     logger.info(f"buckets to delete: {buckets_to_delete}")
+    filename = "s3_bucket_deletion_list.txt"
```
Comment (petr-balogh) [inline ocs_ci/cleanup/aws/cleanup.py:327]: What location are you intend to store this file? This should be somewhere in logs dir I guess.
For what reason we exactly need this file to exist when those buckets suppose to be deleted?

File: ocs_ci/cleanup/aws/cleanup.py [hunk 2]
```diff
@@ -324,9 +324,12 @@ def delete_buckets(bucket_prefix, hours):
...
+    with open(filename, "w") as f:
+        f.write(str(buckets_to_delete))
     for bucket in buckets_to_delete:
         logger.info(f"Delete bucket {bucket}")
-        aws.delete_bucket(bucket)
+        # aws.delete_bucket(bucket)
```
Comment (petr-balogh) [inline ocs_ci/cleanup/aws/cleanup.py:332]: I guess the comment was just for testing?

File: ocs_ci/cleanup/aws/cleanup.py [hunk 3]
```diff
@@ -324,9 +324,12 @@ def delete_buckets(bucket_prefix, hours):
     aws = AWS()
     buckets_to_delete = aws.get_buckets_to_delete(bucket_prefix, hours)
     logger.info(f"buckets to delete: {buckets_to_delete}")
+    filename = "s3_bucket_deletion_list.txt"
+    with open(filename, "w") as f:
```
Comment (petr-balogh) [inline ocs_ci/cleanup/aws/cleanup.py:328]: should we mabye use `"a"`
To append to file?

Not sure if this function will be called multiple time for some reason in future you will overwrite what was already written in file.

File: ocs_ci/cleanup/aws/cleanup.py (hunk 2 above)
Comment (OdedViner) [reply ocs_ci/cleanup/aws/cleanup.py:332]: correct.. I wanted to check the content of the file

File: ocs_ci/cleanup/aws/cleanup.py (hunk 1 above)
Comment (OdedViner) [reply ocs_ci/cleanup/aws/cleanup.py:327]: What location are you intend to store this file? This should be somewhere in logs dir I guess. ->
the location is `~`, I copied the code from here:
https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/cleanup/aws/cleanup.py#L458


For what reason we exactly need this file to exist when those buckets suppose to be deleted?
I want to check who buckets deleted. I cant see it via `log.info`

File: ocs_ci/cleanup/aws/cleanup.py (hunk 3 above)
Comment (OdedViner) [reply ocs_ci/cleanup/aws/cleanup.py:328]: I copied the code from here:
https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/cleanup/aws/cleanup.py#L458

Comment (OdedViner) [reply ocs_ci/cleanup/aws/cleanup.py:328]: CC @clacroix12

**Pull request closed without merging**